File: Beecrowd/Beginner/1000 - 1099/1010 - Simple Calculate/Python/1010_SimpleCalculate.py
input1 = input().split(" ")
input2 = input().split(" ")
product1Code, product1Unii, product1Price = input1
product2Code, product2Unii, product2Price = input2

total = (int(product1Unii) * float(product1Price)) + (int(product2Unii) * float(product2Price))

print(f"VALOR A PAGAR: R$ {total:.2f}")

File: Beecrowd/Beginner/1000 - 1099/1010 - Simple Calculate/C++/1010_SimpleCalculate.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
    int product1Code, product1Units, product2Code, product2Units;
    double product1Price, product2Price;

    cin >> product1Code >> product1Units >> product1Price;
    cin >> product2Code >> product2Units >> product2Price;

    double total = product1Units * product1Price + product2Units * product2Price;

    cout << fixed << setprecision(2) << "VALOR A PAGAR: R$ " << total << endl;
    
    return 0;
}